quality-measure-engine.gemspec

```diff
@@ -17,12 +17,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'mongoid', '~> 5.0.0'
+  gem.add_dependency 'codeclimate-test-reporter', '0.6.0'
```

Comment:
no... when i bundle updated it updated to a version that required changes (like config type changes). I deleted the dependency specification.